### ISSUE TITLE
Invest in Sabzimatic  ?

### ISSUE THREAD

**Issue by mink007** [Bloomberg-Beta/Manual#113]
Hello,

I am founder of Sabzimatic, working on design and development of cooking robot for household.

Here is the little bit about me and the problem. Play at 2x speed if i am slow in my speech.
https://www.youtube.com/watch?v=v4SgztRrYEA

patent pending on the invention. 

website is www.sabzimatic.com and you can scroll down for some of the developments of the product.

Happy to have a zoom video call and can go over the invention, business and discuss if you have any questions.

Cheers,
mandeep
https://www.linkedin.com/in/smandeep/

**Comment by mink007** [comment]
based out of Portland Oregon - north of California.

**Comment by loriberenberg** [comment]
Hi Mandeep, apologies for the delay here. Unfortunately because we invest in the future of work, a cooking robot will be out of scope for us. Wishing you the best. Lori
